PROJECT

```diff
@@ -148,4 +148,30 @@ resources:
     defaulting: true
     validation: true
     webhookVersion: v1
+- api:
+    crdVersion: v1
+    namespaced: true
+  controller: true
+  domain: redhat.io
+  group: redhatcop
+  kind: RabbitMQSecretEngineRole
```

Comment:
soemthing does not look right here as RabbitMQSecretEngineRole appears twice

Reply:
Fixed.